src/app/project/project-details-tab/project-details-tab.component.ts

```diff
@@ -14,6 +16,8 @@ import * as _ from 'lodash';
 // need to import leaflet this way to include the shapefile->geojson plugin
 declare let L;
 const encode = encodeURIComponent;
+L.esri = Esri as any;  // Manually attach Esri to L
```

Comment:
Was an effort made to try to use the right type here? I think it would just consist of downloading the right package and finding the right type to use: https://www.npmjs.com/package/@types/esri-leaflet

Reply:
All of the Esri imports have been optimized in the next commit, and all instances of "any" were removed.